distributed/comm/tests/test_ucx.py

```diff
@@ -4,12 +4,12 @@
 
```

Comment:
This whole module is broken at least against the latest version of ucp. It needs bugfixing and CI integration.

Reply:
cc @quasiben

Reply:
cc also @pentschev

Reply:
> This whole module is broken at least against the latest version of ucp. It needs bugfixing and CI integration.

Could you provide more details? How are you installing UCX-Py and UCX, and how are you launching the tests? What errors do you see?

We run those tests in https://github.com/rapidsai/ucx-py/blob/branch-0.21/ci/gpu/build.sh#L111 and they're currently passing [here](https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/rapidsai%2Fgpuci%2Fucx-py%2Fprb%2Fucx-py-gpu-build/detail/ucx-py-gpu-build/3122/pipeline/) where we test both UCX 1.9 and master branch.

As for CI integration, this is in the scope of https://github.com/dask/community/issues/138 .

Reply:
Apologies, I botched the packages installation. Should've RTFM.

Reply:
No worries, thanks for the update.

From the UCX side I can confirm these changes still work, so I'm approving on behalf of UCX changes only and leave other changes for those with better understanding than myself to judge.